.gitignore

```diff
@@ -1,4 +1,30 @@
-.idea/
+*.class
+*.idea
+.settings
```

Comment:
Should we remove this one as well? Since it is covered by line 18, 21 and 25?

Reply:
@rbuisson isn't it be better if we remove line 18, 21 and 25? As all the **.settings** folder/file will be taken care by line 3

Reply:
I thought it would be better to be more specific in what files we exclude, to avoid that a '.settings' file that is actually in the sources willingly would not be ignored.
But I have no strong opinion on that point.

Reply:
same here as well 😄